NuRadioMC/simulation/simulation2.py

```diff
@@ -141,7 +150,7 @@ def __init__(self, eventlist,
         else:
             self._bandwidth = bandwidth
         self._Vrms = (self._Tnoise * 50 * constants.k *
-                       self._bandwidth / units.Hz) ** 0.5
+                       self._bandwidth / units.Hz) ** 0.5  # from elog:1566
```

Comment:
What is this comment tyring to tell me? This will be very unclear, if we move away from ARIANNA elog.

Reply:
I stubled accross this equation and didn't understand where it came from, and it took me a while to figure out where it came from. I agree that the reference is not optimal but better than nothing.